versioned_docs/version-8.2/reference/release-policy.md

```diff
@@ -55,4 +57,4 @@ If you are not an enterprise customer, you can stay up to date via [release blog
 
 ### New versions
 
-If you are running Camunda 8 Self-Managed, follow our [update guide](/guides/update-guide/introduction.md).
+If you are running Camunda 8 Self-Managed, follow our [update guide](/self-managed/operational-guides/update-guide/introduction.md).
```

Comment:
This update guide had not been migrated over yet, so maintaining the link here, and in `8.1`.